ERCS/erc-x.md

```diff
@@ -0,0 +1,89 @@
+---
+eip: TBD
```

Comment:
```suggestion
eip: 7729
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.